projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order/address-book.ts

```diff
@@ -0,0 +1,17 @@
+import { checkAllElements, TabElement } from '../tabbing-order';
+import { fillShippingAddress } from '../../checkout-forms';
+import { user } from '../../../sample-data/checkout-flow';
+
+export function addressBookAddAddressTabbingOrder(config: TabElement[]) {
+  cy.visit('/my-account/address-book');
+  fillShippingAddress(user);
+
+  cy.contains('Add new address').click();
```

Comment:
I'm not sure if that's the best way. Can we go with `cy.get('...').contains('...')` - it might be safer to use that? Correct me if I'm wrong